### DISCUSSION TITLE
how to pxrUsd to mayaUsd?

### DISCUSSION THREAD

**Discussion by cds1503** [Autodesk/maya-usd#1850] [Q&A]
Hello, I am a user who used pxrUsd in Maya.

As I use maya 2022, I have a few questions, so I would like to ask you.

1. Isn't it recommended to use the pxrUsd plugin?
2. If No.1 is No, How can I export mayaUsdProxyShape as references? (like pxrUsdReferenceAssembly)
3. I am trying to scatter by instance using mash. When pxrUsdReferenceAssembly is used as a Source and Scatter is performed, the Instance Source is exported as a Reference. How do I proceed with mayaUsd like this?

4. If No.1 is Yes, In the pxrUsdReferenceAssembly node, the filePath is missing from the AETemplate. And the File Browser Button is gone. how do i solve it??

Best Regard,
Thank you

**Comment by brycegbrazen** [comment]
Just putting in my two cents, 

1. I believe the pxrUsd plugin is deprecated in favor of mayaUsd.
2. I believe the way USD works is that you load in an exported USD file on disk as a reference into your stage. Therefore, there is no such thing as exporting the proxy shape as a reference. I would recommend reading over the USD glossary of terms. It should hopefully clear lots of these kind of things up. https://graphics.pixar.com/usd/release/glossary.html#usdglossary-references

If anything I've said is incorrect anyone feel free to correct me 😄

**Reply by cds1503** [reply]
Thank you reply

I've been developing usd for a long time and read glossary too.
I think the direction I am pursuing is too focused on geometry export.
I think that I can export 3 objects together with proxyShape.
However, if proxyShape cannot be exported, you need to add reference to each one by script (no problem). However, it is too inefficient and the impact of TD is too great.
So I hope that the export format in the existing pxrUsd is also supported.
